boards/riscv/tlsr9518adk80d/tlsr9518adk80d_defconfig

```diff
@@ -4,9 +4,11 @@
 CONFIG_SOC_SERIES_RISCV_TELINK_B91=y
```

Comment:
Maybe add in the commit message why this change?

Reply:
Done.